projects/orquestra-workflow-runtime/tests/data/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
What .json files do you expect here, needing to be ignored?

Reply:
Adding comment in the .gitignore